app/models/Movement.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.libs.json.{Json, OFormat}
+
+import java.time.LocalDateTime
+
+case class Movement(id: String, messageCode: String, timestamp: LocalDateTime, office: String)
```

Comment:
Maybe this `id` should be called `messageSender` to match the header name?

Reply:
Reason it's called id is because from our service's perspective that's what it is. An identifier to join up the outbound declarations with the inbound responses from NCTS core.

Reply:
Ah cool, so you are using the message sender header value to track both sides of the request

Reply:
Yes that's correct